processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -111,6 +112,17 @@ public BitmapFactory getBitmapFactory()
   }
 
   @Override
+  public BitmapIndex getBitmapIndex(String dimension)
```

Comment:
ok, I'm fine with this PR as-is, that rename if desired could be handled in later patch